.gitignore

```diff
@@ -1,2 +1,6 @@
 coverage.*
-bin/
\ No newline at end of file
+bin/
+
+# Jetbrain IED
+.idea
+*.iml
```

Comment:
Hello, I don't think we should ignore these things in specific projects. If you don't want to mess up the code base, you can ignore these editor and IDE things globally.
For example:
```
$ git config --global core.excludesfile ~/.gitignore_global
```
Then you can ignore these things in `~/.gitignore_global`